tests/test_input_tool.py

```diff
@@ -0,0 +1,252 @@
+from evagram.database import input_tool
+import sys
+import unittest
+import psycopg2
+from dotenv import load_dotenv
+import os
+
+
+conn = psycopg2.connect("host=localhost port=5432 dbname=plots user=postgres")
+wf_dictionary = {
+    'owner': 'postgres',
+    'experiment': 'experiment1',
+    'eva_directory': 'tests/eva'
+}
+
+
+class TestDatabaseInputTool(unittest.TestCase):
+    def setUp(self):
+        self.cur = conn.cursor()
+        input_tool.Session(conn, wf_dictionary)
+
+        self.cur.execute(
+            """SELECT setval('owners_owner_id_seq',
+            (SELECT MAX(owner_id) FROM owners)+1)""")
+        self.cur.execute(
+            """SELECT setval('experiments_experiment_id_seq',
+            (SELECT MAX(experiment_id) FROM experiments)+1)""")
+        self.cur.execute(
+            """SELECT setval('plots_plot_id_seq',
+            (SELECT MAX(plot_id) FROM plots)+1)""")
+        self.cur.execute(
+            """SELECT setval('observations_observation_id_seq',
+            (SELECT MAX(observation_id) FROM observations)+1)""")
+
+    def tearDown(self):
+        conn.rollback()
+        self.cur.close()
+
```

Comment:
It's unclear whether we should keep all these tests in evagram_input. We'll have to think about this more. The purpose of this repository was to be a lightweight dependency for swell, it would be nice to have it not keep around large pkl files